dotcom-rendering/src/web/components/MostViewed/MostViewedFooter/MostViewedFooterData.tsx

```diff
@@ -72,6 +72,7 @@ export const MostViewedFooterData = ({
 				css={css`
 					width: 100%;
 				`}
+				data-cy="mostviewed-footer"
```

Comment:
I've added this test attribute so that I can assert that this content is _not_ in the dom prior to scrolling the page

Reply:
Do we use `getByText` with the heading text for selectors like this if we wanted to follow the ["How would users find it?"](https://testing-library.com/docs/queries/about#priority) methodology.

Reply:
For Cypress tests we follow the [recommendations](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) and use `data-cy` selectors.

Swings and roundabouts but I do think it's nice to separate the hooks you're using for selection away from how you implement the product you're testing. But it's true that selecting by text can feel and read better, when it works.